src/tests/JIT/opt/ObjectStackAllocation/ObjectStackAllocationTests.csproj

```
@@ -16,9 +16,5 @@
     <CLRTestEnvironmentVariable Include="DOTNET_JitNoForceFallback" Value="1" />
```

Why is `DOTNET_JitNoForceFallback=0` necessary? Can it hide bugs?